### DISCUSSION TITLE
Article 20: Irregular Termination of Citizenship | Citizenship Law

### DISCUSSION THREAD

**Discussion by DiegoCavalcante97** [liberland/laws#42] [Ideas]
Article 20 states that 'The Citizenship Commission shall terminate the Citizenship of a Citizen who is in serious breach of their Citizenship duties, for which the contract stipulates the sanction of an irregular termination of their Citizenship.'

I have some concerns about the potential for abuse of this article, as it could be employed for political persecution or targeting dissidents. It would be crucial to have clear and specific criteria for the revocation of citizenship.

Besides, wouldn't it be better to simply revoke the political rights? 

I also have some questions regarding the status of a Liberland resident following citizenship loss: Are they barred from the country? Do they retain property rights? Can a non-citizen still reside in Liberland?

Given the potential risks associated with Article 20, I find it essential to address these questions within the legal framework.

**Comment by omegaavocats** [comment]
I have considerable reservations regarding the misuse of this item, as it might be used to persecute political opponents or single out dissidents. Clear and precise standards for citizenship revocation would be essential.

Furthermore, wouldn't it be preferable to just abolish the political rights?

I also have some inquiries on the situation of a Liberland resident who has lost their citizenship: Are they denied entry to the nation? Do they still have ownership rights? Can a person who is not a citizen live in [Paris?
](https://avocat-succession.omega-avocats.fr/)
Considering the dangers that could result from
